v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -209,5 +210,13 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.0/pca/gnomad.v4.0.pca_loadings.ht',
         },
     },
+    ReferenceDataset.mitomap: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.MITO]),
+            VERSION: '1.0',
+            # Downloaded via https://www.mitomap.org/foswiki/bin/view/MITOMAP/ConfirmedMutations
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/mitochondrial/MITOMAP/mitomap_confirmed_mutations_nov_2024.csv',
```

Comment:
I just downloaded this, the last version we have was from 2022. Looks like there's about 30 more pathogenic variants.

Reply:
yay newer data!

Reply:
Why are we downloading this and storing a copy in seqr-reference-data instead of directly using the external download?

Reply:
The site has a button to download the csv but I found no way to programmatically download via an api, I could be wrong though https://www.mitomap.org/foswiki/bin/view/MITOMAP/ConfirmedMutations

Reply:
Ah!  It looks like the site generates temporary links when you click the "CSV" button.  For examples: blob:https://www.mitomap.org/0f70e9e7-b27b-4db6-9026-014578cdac39... These appear to expire quickly so yes, hosting our own is probably the right idea here.